index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Homework1</title>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+  </head>
+  <body>
+      <header>
+          <nav>
+              <ul>
+                  <li><a href="">Home</a></li>
+                  <li><a href="">About me</a></li>
+                  <li><a class="active" href="">Base page</a></li>
+              </ul>         
+          </nav>
+      </header>
+      <center>
+      <div class="main">
+          <h2>What do I miss about Munich now when in isolation?</h2>
+          <section class="card">
+              <h2 class="card_title">Nordbad</h2>
+              <div class="container"></div>
+              <div class="card_imgWrapper">
+                   <img class="card_img" src="nordbad.jpg">
+              </div>
+              <div class="card_description"><br>Nordbad is a fun and relaxing place where to go on a weekend </br>when there's bad weather and you find it <a href="https://www.google.com/maps/place/Nordbad/@48.1603281,11.5615484,17z/data=!3m1!4b1!4m5!3m4!1s0x479e75deb97b8337:0x6bca64d0f3009f64!8m2!3d48.1603245!4d11.5637424">here</a>
+              </div>
+          </section>
+
+          <section class="card">
+            <h2 class="card_title">Skiing nearby Munich</h2>
```

Comment:
I also miss skiing 
<img width="1452" alt="Screenshot 2020-04-19 at 11 31 39" src="https://user-images.githubusercontent.com/13404719/79684418-871c2b00-8231-11ea-8116-908d4a6352cf.png">

Reply:
:) I know from your website (I checked out last night and it's cool :) that you are an avid skier and climber! Any tips on where to go climb in the nature around Munich in these times? Thanks